coq.opam

```diff
@@ -6,13 +6,13 @@ bug-reports: "https://github.com/coq/coq/issues"
 dev-repo: "https://github.com/coq/coq.git"
 license: "LGPL-2.1"
 
-available: [ ocaml-version >= "4.02.3" ]
+available: [ ocaml-version >= "4.05.0" ]
 
 depends: [
-  "dune"      { build }
+  "dune"      { build & >= 1.2.0 }
   "ocamlfind" { build }
   "num"
-  "camlp5"
+  "camlp5"    {         >= 7.03  }
```

Comment:
Aren't we accepting 7.01?

Reply:
IIRC 7.02 has a critical bug making Coq fail so I'd rather play on the safe side here.

Reply:
Or indeed maybe it was 7.02 the one with the bug, hard to tell....

Reply:
Maybe the problem was with `Bytes`. Umm, I think I'm gonna up the version to 7.02 indeed.

Reply:
Ok, I've settled in 7.02. I think that 7.01 had troubles working with 4.06.1 in the context of Coq, so IMHO that should be fine; we can keep 7.01 + 4.05.0 in the CI, but for OPAM it will save us headaches to move to 7.02.